src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandler.java

```diff
@@ -1,15 +1,16 @@
 
 package com.blazemeter.jmeter.http2.sampler;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
+import java.io.*;
```

Comment:
What do you think if we just import what we need? (instead of the whole library :D)